include/proto/kvs.proto

```diff
@@ -70,7 +70,7 @@ message KeyTuple {
 message KeyRequest {
   required RequestType type = 1;
   repeated KeyTuple tuples = 2;
-  optional string response_address = 3;
+  required string response_address = 3;
```

Comment:
Why are we changing this to `required`? I remember in KVS gossip the `response_address` is left empty because we don't need response?

Reply:
Oh, I see. I changed it to `required` because user requests always require a response. We can always just put an empty value in for the gossip protocol.